src/integration/java/io/pinecone/integration/controlPlane/pod/CreateDescribeListAndDeleteIndexTest.java

```diff
@@ -1,4 +1,4 @@
-package io.pinecone.integration.controlPlane.index.pod;
+package io.pinecone.integration.controlPlane.pod;
```

Comment:
I don't think we need the extra `index` here, it would only be index and collections anyways.

Reply:
Oh I meant to have two directories:
1. index tests
2. collection tests

So I created a placeholder but no big deal

Reply:
Right, yeah I think we just don't need the extra level given collections could be classified as index operations. I think we can just group everything under `pod`/`serverless` for now.